src/logic/shared/saveCurrentProject.js

```diff
@@ -0,0 +1,17 @@
+import {
+  getCurrentProject,
+  getCurrentUserId,
+} from '../../selectors';
+import {isPristineProject} from '../../util/projectUtils';
+import {saveProject} from '../../clients/firebase';
+import {projectSuccessfullySaved} from '../../actions/projects';
+
+export async function saveCurrentProject(state) {
```

Comment:
Probably can make this a default export?

Reply:
And also maybe make an index module that reexports it?

Reply:
Had trouble with linting the index module so just changed to default